AllReadyApp/Web-App/AllReady.UnitTest/Controllers/ManageControllerTests.cs

```diff
@@ -899,14 +899,28 @@ public void DisableTwoFactorAuthenticationHasValidateAntiForgeryTokenAttribute()
 
         }
 
-        [Fact(Skip = "NotImplemented")]
+        [Fact]
         public void ChangePasswordGetReturnsAView()
-        {
+        {
+            var controller = new ManageController(null, null, null);
+
+            //Act
+            var result = controller.ChangePassword();
+
+            //Assert
+            Assert.IsType(typeof(ViewResult), result);

```

Comment:
Hi, assume you mean use the generic version instead? i.e  Assert.IsType<ChangeEmailViewModel>(result);